src/lib/nlcDb.js

```diff
@@ -212,8 +222,12 @@ module.exports.post = function(classification, type, selectedClass) {
 			doc.classification = classification;
 		}
 
-		if (selectedClass)
+		if (selectedClass) {
 			doc.selectedClass = selectedClass;
+			if (truthy(process.env.HUBOT_WATSON_NLC_AUTO_APPROVE)) {
+				doc.approved = true;
```

Comment:
Need to confirm with @normanb , but I believe the UI sets `doc.approved` to a timestamp indicating when the document was approved.